app/src/lib/components/HunkViewer.svelte

```diff
@@ -75,7 +73,7 @@
 		class:opacity-60={section.hunk.locked && !isFileLocked}
 	>
 		<div bind:this={contents} class="hunk__bg-stretch">
-			{#if linesModified > 1000 && !alwaysShow}
+			{#if linesModified > 5000 && !alwaysShow}
```

Comment:
@mtsgrd any opinion on what we should set this to? I had a ~4900 line test file that took ~2-3s to render now. Still seems a bit long though, so maybe we set this to 2500-3000, somewhere in that range?

Reply:
i guess 2500 is reasonable

Reply:
Updated to 2500 :+1: